Gemfile

```diff
@@ -1,25 +1,32 @@
 source 'https://rubygems.org'
-
-
+gem 'pry'
+gem 'rest-client'
+
+# Views interpreter
+gem 'haml'
+# TODO: findout what all these do
+gem 'erubis'
+# User Authentefication
+gem 'devise', '4.3.0'
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
```

Comment:
a good practice is to remove all comments, you don't really need them and the file looks cleaner without these comments